dept44-build-tools/src/main/java/se/sundsvall/dept44/maven/enforcer/rule/Dept44RequiredPropertiesRule.java

```diff
@@ -113,7 +115,7 @@ private List<String> ensureOpenApiPropertiesAreSet(final String resourcesDir) th
 					}
 				}
 			} catch (IOException e) {
-				throw new EnforcerRuleException("Unable to load properties/YAML file " + propertyFile.getName(), e);
+				throw new EnforcerRuleException("Unable to load properties/YAML file " + propertyFile.getName() + ": " + e.getMessage(), e);
```

Comment:
String.format instead of the string concatenation?

Reply:
Yesbox. Self clear